### DISCUSSION TITLE
Off to 2025

### DISCUSSION THREAD

**Discussion by giscus** [aCursedComrade/acursedcomrade.github.io#20] [General]
# Off to 2025

Well… this blog has been dry for a real long time huh… >_> Anyways, it’s time to leave 2024 behind and jump head first into 2025 which also means it’s time for me to “step out from the shadows”

https://comradelab.win/2024/12/into-2025/

**Comment by nimash3eshan** [comment]
What an inspiring post to kick off 2025! 🎉 Huge congratulations on graduating with first-class honors. that’s an incredible achievement and a testament to your hard work and passion for computer security. It’s amazing to see how far you’ve come. cheers to new beginnings! 🚀

**Reply by aCursedComrade** [reply]
That's a little sugary for you but thanks, Nimz. Rooting for you too :)
